slices/erlang-mnesia.yaml

```diff
@@ -0,0 +1,15 @@
+package: erlang-mnesia
+
+essential:
+  - erlang-mnesia_copyright
+
+slices:
+  modules:
+    essential:
+      - erlang-base_bins
```

Comment:
yeah they require the bins slice too because this is where the erl compiler and other utilities like escript are found. Without those, the module files alone are useless.